kcidb/db/postgresql/v04_09.py

```diff
@@ -0,0 +1,165 @@
+"""Kernel CI report database - PostgreSQL schema v4.8"""
+
+import textwrap
+from kcidb.misc import merge_dicts
+import kcidb.io as io
+from kcidb.db.postgresql.schema import \
+    TextColumn, TextArrayColumn, JSONColumn, \
+    BoolColumn, TimestampColumn, Table, Index
+from .v04_08 import Schema as PreviousSchema
+
+
+CREATE_UNIT_PREFIX_TYPE_STATEMENT = textwrap.dedent("""\
+    DO $$ BEGIN
+        CREATE TYPE UNIT_PREFIX AS ENUM ('metric', 'binary');
+    EXCEPTION
+        WHEN duplicate_object THEN null;
+    END $$
+""")
```

Comment:
We don't need this here.

Reply:
Yes, it was for `number.prefix`. Dropped it.